src/components/Login/Login.js

```diff
@@ -2,8 +2,9 @@ import React, { useState } from "react";
 import { useNavigate } from "react-router-dom";
 import axios from "axios";
 import Cookies from "js-cookie";
+import { ReactComponent as ErrorIconRed } from "../../icons/error-icon-red.svg";
```

Comment:
no need for the import here as this is already part of the `<Error />` component